package.json

```diff
@@ -7,16 +7,15 @@
   "contributors": [
     "Ricardo Gobbo de Souza <ricardogobbosouza@yahoo.com.br>"
   ],
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
   "files": [
-    "lib",
-    "types"
+    "dist"
   ],
-  "main": "lib/index.js",
-  "types": "types/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "lint": "eslint --ext .js,.vue .",
-    "release": "yarn test && standard-version && git push --follow-tags && npm publish",
+    "lint": "eslint --ext .ts .",
+    "release": "yarn test && yarn build && standard-version && git push --follow-tags && npm publish",
```

Comment:
Neat: Probably not needed for us but doing `build` after version, ensures if bundling version, we use updated one